DiagManager/CustomDiagnostics/SQL Base/MiscPssdiagInfo.sql

```diff
@@ -73,150 +73,42 @@ PRINT ''
-
-	EXEC xp_instance_regread
-		@rootkey = @myhive,
-		@key = @mykey,
-		@value_name = 'InstallDate',
-		@value = @regvalueint OUTPUT
-
-	INSERT INTO #summary VALUES ('operating system install date', CONVERT(VARCHAR(23), DATEADD(SECOND, @regvalueint, '1970-01-01'), 121))
-
-	/*
-	other possible values of interest
-	CurrentType
-	InstallationType
-	EditionID
-	SoftwareType
-	*/
-END
+	INSERT INTO #summary VALUES ('operating system', NULL)	
 
-IF @fWinVista = 1
-BEGIN
-	SET @mykey = N'SYSTEM\CurrentControlSet\Control\SystemInformation'
-	--get system info from registry
-	EXEC xp_instance_regread
-		@rootkey = @myhive,
-		@key = @mykey,
-		@value_name = 'SystemManufacturer',
-		@value = @regvalue OUTPUT
-	INSERT INTO #summary VALUES ('registry SystemManufacturer', @regvalue)
-	EXEC xp_instance_regread
-		@rootkey = @myhive,
-		@key = @mykey,
-		@value_name = 'SystemProductName',
-		@value = @regvalue OUTPUT
-	INSERT INTO #summary VALUES ('registry SystemProductName', @regvalue)	
-
-	--get powerplan from registry
-	SET @mykey = N'SYSTEM\CurrentControlSet\Control\Power\User\Default\PowerSchemes'
-
-	EXEC   xp_instance_regread
-		@rootkey = @myhive,
-		@key = @mykey,
-		@value_name = 'ActivePowerScheme',
-		@value = @regvalue OUTPUT
-
-	SET @mykey = @mykey + N'\' + @regvalue
-
-	EXEC   xp_instance_regread
-		@rootkey = @myhive,
-		@key = @mykey,
-		@value_name = 'FriendlyName',
-		@value = @regvalue OUTPUT
-
-	DECLARE @string NVARCHAR(100)
-	SET @string = N'@%SystemRoot%\system32\powrprof.dll,'
-	SET @pos = CHARINDEX(@string, @regvalue)
-	IF @pos != 0
-	BEGIN
-		DECLARE @len int
-		SET @len = LEN(@string)+1
-		SET @pos = CHARINDEX(N',', @regvalue, @len)
-		IF @pos != 0
-		BEGIN
-			SET @regvalue = SUBSTRING(@regvalue, @pos+1, LEN(@regvalue))
-		END
-	END
-
-	INSERT INTO #summary VALUES ('registry ActivePowerScheme (default)', @regvalue)	
-
-	SET @mykey = N'SYSTEM\CurrentControlSet\Control\Power\User\PowerSchemes'
-
-	EXEC   xp_instance_regread
-		@rootkey = 'HKEY_LOCAL_MACHINE',
-		@key = @mykey,
-		@value_name = 'ActivePowerScheme',
-		@value = @regvalue OUTPUT
-
-	SET @mykey = @mykey + N'\' + @regvalue
-
-	EXEC   xp_instance_regread
-		@rootkey = @myhive,
-		@key = @mykey,
-		@value_name = 'FriendlyName',
-		@value = @regvalue OUTPUT
-
-	INSERT INTO #summary VALUES ('registry ActivePowerScheme', @regvalue)	
+	INSERT INTO #summary VALUES ('operating system install date',NULL)
+ 
 END
+ 
+	--inserting NULL to keep same #summary structure 
+	INSERT INTO #summary VALUES ('registry SystemManufacturer', NULL)
+
+	INSERT INTO #summary VALUES ('registry SystemProductName', NULL)	
+
+	INSERT INTO #summary VALUES ('registry ActivePowerScheme (default)', NULL)	
+
+	INSERT INTO #summary VALUES ('registry ActivePowerScheme', NULL)	
+
+	--inserting OS Edition and Build from @@Version 
+	INSERT INTO #summary VALUES ('OS Edition and Build from @@Version',  LTRIM(SUBSTRING(@@VERSION,CHARINDEX(' on ',@@VERSION)+3,100)))
```

Comment:
@hacitandogan when pssdiag generates the raw file, I am seeing an empty line, like this. Do you have any idea why?

<img width="608" alt="image" src="https://user-images.githubusercontent.com/23519517/163010621-5cf52121-aa0f-4468-89e5-3731ac19dde4.png">

Reply:
@PiJoCoder , I see @@VERSION has a line feed at the end (ASCII 10) , we can clear it before inserting into table.
```suggestion
	INSERT INTO #summary VALUES ('OS Edition and Build from @@Version',  REPLACE(LTRIM(SUBSTRING(@@VERSION,CHARINDEX(' on ',@@VERSION)+3,100)),CHAR(10),''))
```